RELEASE_NOTES.md

```diff
@@ -1,3 +1,52 @@
+# ChefDK 4.3.0
+
+## RHEL 8 Support Added
+
+ChefDK 4.3 now includes native packages for RHEL 8 with all builds now validated on RHEL 8 hosts.
+
+## SLES 11 EOL
+
+Packages will no longer be built for SUSE Linux Enterprise Server (SLES) 11 as SLES 11 exited the 'General Support' phase on March 31, 2019. See C[hef's Platform End-of-Life Policy](https://docs.chef.io/platforms.html#platform-end-of-life-policy) for more information on when Chef ends support for an OS release.
+
+## Updated Components
+
+### Chef Infra Client
+
+Chef Infra Client has been updated from 15.1 to 15.2 with new and improved resources and support for RHEL 8. See the [Chef Infra Client 15.2 Release Notes](https://github.com/chef/chef/blob/master/RELEASE_NOTES.md#chef-infra-client-152) for a complete list of new and improved functionality.
```

Comment:
Does this mean you need to update the https://docs.chef.io/release_notes.html page? I would prefer that links refer to the docs.chef.io rather than the repo.

Reply:
I'd link to that if we had it fully updated. It's not at the moment so I'm sorta working with what I have.